### PR TITLE
Update react dependencies 

### PR DESCRIPTION
**Pull request by aurambaj**

### CODE REVIEW

File: webapp/package.json [hunk 1]
```diff
@@ -5,22 +5,24 @@
...
   "dependencies": {
-    "alt": "0.17.1",
-    "bootstrap-sass": "3.3.5",
+    "alt": "^0.18.6",
+    "alt-container": "^1.0.2",
+    "alt-mixins": "^1.0.0",
+    "bootstrap-sass": "^3.3.7",
     "cldr-data": "^28.0.2",
     "cldrjs": "^0.4.3",
-    "history": "1.9.1",
```
Comment (willyau) [inline webapp/package.json]: nice

File: webapp/src/main/resources/public/js/app.js [hunk 2]
```diff
@@ -105,21 +112,21 @@ BaseClient.authenticateHandler = function () {
...
+                    <Modal.Header closeButton={true}>
+                        <Modal.Title>
+                            <FormattedMessage id="error.modal.header.title" />
+                        </Modal.Title>
+                    </Modal.Header>
+                    <Modal.Body>
+                        <FormattedMessage id="error.modal.message.loggedOut" />
+                    </Modal.Body>
+                    <Modal.Footer>
+                        <Button bsStyle="primary" onClick={okOnClick}>Okay</Button>
```
Comment (willyau) [inline webapp/src/main/resources/public/js/app.js]: ha missing FormattedMessage for Okay

File: webapp/src/main/resources/public/js/components/workbench/LocalesDropdown.js [hunk 3]
```diff
@@ -277,7 +277,7 @@ let LocalesDropDown = React.createClass({
 
         return (
                 <span className="mlm locale-dropdown">
-                <DropdownButton title={this.getButtonText()} onToggle={this.onDropdownToggle} open={this.state.isDropdownOpenned}>
+                <DropdownButton id="WorkbenchLocaleDropdown" title={this.getButtonText()} onToggle={this.onDropdownToggle} open={this.state.isDropdownOpenned}>
```
Comment (willyau) [inline webapp/src/main/resources/public/js/components/workbench/LocalesDropdown.js]: what's the id for?

File: webapp/src/main/resources/public/js/sdk/RepositoryClient.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+    getRepositories() {
+        return this.get(this.getUrl(), {});
+    }
+
+    getEntityName() {
+        return 'repositories';
+    }
+}
+;
```
Comment (willyau) [inline webapp/src/main/resources/public/js/sdk/RepositoryClient.js]: semi colon on same line

File: webapp/src/main/resources/public/js/sdk/RepositoryClient.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import BaseClient from "./BaseClient";
+
+class RepositoryClient extends BaseClient {
+
```
Comment (willyau) [inline webapp/src/main/resources/public/js/sdk/RepositoryClient.js]: empty line

File: webapp/src/main/resources/public/js/stores/workbench/SearchParamsStore.js [hunk 6]
```diff
@@ -221,14 +218,29 @@ class SearchParamsStore {
...
+        
+        if (typeof repoIds !== "undefined") {
+            if (Array.isArray(repoIds)) {
+                repoIds = repoIds.map((value) => parseInt(value));
+            } else {
+                repoIds = [parseInt(repoIds)];
+            }
+        }
+        
+        if (typeof bcp47Tags !== "undefined"&& ! Array.isArray(bcp47Tags)) {
```
Comment (willyau) [inline webapp/src/main/resources/public/js/stores/workbench/SearchParamsStore.js]: space after " &&

we can also try: nom add prettier

File: webapp/src/main/resources/public/js/components/workbench/LocalesDropdown.js (hunk 3 above)
Comment (aurambaj) [inline webapp/src/main/resources/public/js/components/workbench/LocalesDropdown.js]: react complain if there is no id

File: webapp/src/main/resources/public/js/sdk/RepositoryClient.js (hunk 4 above)
Comment (aurambaj) [inline webapp/src/main/resources/public/js/sdk/RepositoryClient.js]: done

File: webapp/src/main/resources/public/js/stores/workbench/SearchParamsStore.js (hunk 6 above)
Comment (aurambaj) [inline webapp/src/main/resources/public/js/stores/workbench/SearchParamsStore.js]: done 

"we can also try: nom add prettier" ?

Comment (willyau) [inline webapp/src/main/resources/public/js/stores/workbench/SearchParamsStore.js]: ha. i meant to write

`npm install --save-dev prettier` 

it's pretty cool. auto formats so we don't have to .

**Pull request merged**